contents/blog/best-open-source-analytics-tools.md

```diff
@@ -392,4 +327,57 @@ Superset is ideal for enterprises with experienced in-house data teams. It can h
+
+### 11. Countly
+![Countly - open source analytics tools](../images/blog/open-source-analytics-tools/countly-screenshot.png)
+
+- **GitHub Stars:** 5.3k
+- **Category:** Product analytics
+- **Alternative to:** Mixpanel, Amplitude, LogRocket
+- **Languages:** JavaScript (82%), HTML (9.4%)
+- [Countly on GitHub](https://github.com/Countly/countly-server)
+
+#### What is Countly?
+
+Like PostHog, [Countly](https://count.ly/) offers a wide range of tools to help you build better products. It has a strong focus on mobile and desktop applications, and enables you to bring together both qualitative and quantitative insights. However many features, such as dashboards, funnel analysis, and retention tracking are only available in the enterprise-level product.
+
+#### Who is Countly for?
+Countly is suitable for both small teams and enterprise businesses, especially those which focus on mobile apps. The crash analytics tools make it useful for customer support and engineering teams, too. 
+
+#### Strengths
+- Wide range of analysis tools
+- Focus on mobile and desktop app development
+- Crash analytics to diagnose bugs
+- Run surveys to get qualitative insights
+
+#### Open source license and monetization
+Countly's Community Edition is available under a AGPLv3 license, but it's somewhat limited. It includes versions of its core analytics features, but customer behavior insights (e.g. retention, cohorts, funnels, user paths) are only available in the paid Enterprise Edition. Countly doesn't provide public pricing information.
+
+### 12. Open Web Analytics
+
+![open web analytics](../images/blog/open-source-analytics-tools/owa.png)
+
+- **GitHub Stars:** 2.2k
+- **Category:** Website analytics
+- **Alternative to:** Google Analytics
+- **Languages:** 67% PHP, 23% JavaScript
+- [OWA on GitHub](https://github.com/Open-Web-Analytics/Open-Web-Analytics)
+
+#### What is Open Web Analytics?
+
+[Open Web Analytics](https://www.openwebanalytics.com/) is an entirely free, open-source alternative to Google Analytics. In addition to usual web analytics basics, it includes a basic heatmap feature, and the ability to track clicks on all DOM elements. It's less focused on privacy than others in this list, so you can use OWA to track when users come back, and perform basic campaign and e-commerce tracking. In comparative terms, it's more advanced than Plausible and Umami, but less advanced than Matomo or PostHog, though the interface is somewhat dated. 
+
+#### Who is Open Web Analytics for?
+
+OWA will suit any developer who wants self-hosted analytics, but desires more tracking capabilities than privacy-focused alternatives like Plausible.
+
+#### Strengths
+
+- Simple GDPR compliance
+- Control over your own data
+- Heatmap and DOM element tracking
+- Basic campaign and e-commerce tracking
+
+#### Open source license and monetization
+Open Web Analytics is distributed under the GPL-2.0 license. There are no paid features.
```

Comment:
Some sort of conclusion? "You should choose PostHog!"

Reply:
Again, keeping this light on marketing lines.